### PR TITLE
{tools}[foss/2021b] MoviePy v1.0.3 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@sassy-crick I think you forgot the patch...

**Comment by sassy-crick** [comment]
> @sassy-crick I think you forgot the patch...

No, that patch is in an already merged version (PR #16479), I only needed to sync it. 
Thanks for reminding me. :-)

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/4a81268a2d9d97281ecc85981319ceb1 for a full test report.

File: easybuild/easyconfigs/m/MoviePy/MoviePy-1.0.3-foss-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    ('proglog', '0.1.10', {
+        'checksums': ['658c28c9c82e4caeb2f25f488fff9ceace22f8d69b15d0c1c86d64275e4ddab4'],
+    }),
+    (name, version, {
+        'source_tmpl': 'moviepy-%(version)s.tar.gz',
+        'patches': ['moviepy-1.0.3-setup.py.patch'],
+        'checksums': [('2884e35d1788077db3ff89e763c5ba7bfddbd7ae9108c9bc809e7ba58fa433f5'),
+                      ('beca82ff8dfbc9cd0b97296f310de3ddfc43e2e49344a32375d4438485c3f3df')],
+    }),
+]
```
Comment (verdurin) [inline easybuild/easyconfigs/m/MoviePy/MoviePy-1.0.3-foss-2021b.eb:40]: @sassy-crick is there a simple `sanity_check_command` you could add?

Comment (sassy-crick) [reply easybuild/easyconfigs/m/MoviePy/MoviePy-1.0.3-foss-2021b.eb:40]: I don't think there is a binary as such, so I added the `sanity_check_paths` instead. 
However, we might want to update PR #16479 as well if we are including this here.

File: easybuild/easyconfigs/m/MoviePy/MoviePy-1.0.3-foss-2021b.eb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+sanity_check_paths = {
```
Comment (casparvl) [inline easybuild/easyconfigs/m/MoviePy/MoviePy-1.0.3-foss-2021b.eb:42]: This path is already the default, see https://github.com/easybuilders/easybuild-easyblocks/blob/839cdea71872022ec9d6917870aeecf4d271eb48/easybuild/easyblocks/generic/pythonbundle.py#L158

So, I propose to remove it again (sorry... :))

**Review by casparvl — CHANGES_REQUESTED** [review]

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
tcn251.local.snellius.surf.nl - Linux Rocky Linux 8.7, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/ca7db40fc963e6b7ec07e2a056a78efc for a full test report.

**Review by casparvl — APPROVED** [review]
Lgtm!

**Comment by casparvl** [comment]
Going in, thanks @sassy-crick!

**Comment by casparvl** [comment]
Going in, thanks @sassy-crick!

**Comment by casparvl** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**